lib/URLButton.js

```diff
@@ -0,0 +1,130 @@
+//reimplimentation of add-on sdk button/toggle, found at
```

Comment:
Also, I know I said we couldn't do this before, but maybe we should look harder at importing webExtension functionality.